app/celer-sim/Transporter.cc

```diff
@@ -105,9 +105,6 @@ auto Transporter<M>::operator()(SpanConstPrimary primaries)
     if (store_track_counts_)
     {
         append_track_counts(track_counts);
-    }
-    if (num_streams_ == 1)
-    {
         result.step_times.push_back(get_step_time());
```

Comment:
Do you think it's useful to have the step times for each thread? Maybe storing the event times would be more useful?

Reply:
Good point... I was thinking `store_track_counts` would be the switch between "minimal output" and "all the output". Event times would be more useful indeed...

Reply:
I've just added a new option to switch off step times (usually we'll want to be looking at these on GPU anyway).

Reply:
There's one more place we'll have to update this below.